test/studies/champs/sub_test

```diff
@@ -93,7 +93,14 @@ if [ -z "$CHAMPS_QUICKSTART" ] || [ ! -z "$CHAMPS_COMPILE_ALL_EXECS" ] ; then
   test_compile geo
   test_compile thermo
   test_compile postLink
-  test_compile post
+
+  # we are seeing OOM issues with the C backend that we attribute to a
+  # system/backend issue. Until we figure out what's going wrong, stop testing
+  # this executable with the C backend
+  if [ "${CHPL_LLVM}" != "none" ] ; then
```

Comment:
Is there a concern if `CHPL_LLVM` is empty/unset?

Reply:
Probably not seriously. Same check is used above to do an LLVM-specific patch. We've had that check probably since the very first iteration of this testing infrastructure.